### ISSUE TITLE
fetch() with no arguments check does an ajax call in safari technology preview

### ISSUE THREAD

**Issue by pjordaan** [angular/zone.js#635]
Recently we ran into lots of 404's with 'undefined' added at the end of the url, but no complaining customers. We found out all came from the latest Safari version and that the line in zone.ts:
```typescript
try {
        // In MS Edge this throws
        fetchPromise = global['fetch']();
      } catch (e) {
```
was causing the ajax call. If I do window.fetch() i a rejected promise in firefox and chrome and in the latest Safari I get an actual 404 ajax call to 'undefined'. Can this be patched so it does not do an actual ajax call in Safari?

**Comment by JiaLiPassion** [comment]
This has been fixed in #622  , please wait for the next release.

**Comment by yannickl88** [comment]
When will there be a next release?

**Comment by JiaLiPassion** [comment]
@yannickl88 , now 0.7.7 is released, you can check it.

**Comment by JiaLiPassion** [comment]
@pjordaan , this one has been fixed and released in 0.7.7, could you check and close this issue?

**Comment by JiaLiPassion** [comment]
close for fixed.
